README.md

```diff
@@ -26,7 +26,22 @@
           "app_category_type": "public.app-category.productivity"
         },
         "linux": {
-          "icon": "resources/linux/<your-project-id>.png"
+          "icon": "resources/linux/<your-project-id>.png",
+          "debian_config": {
```

Comment:
Ah I see where the confusion came from in this PR, forgot to standardise everything to `debian_config` et al, will fix that how.